_includes/venue.html

```diff
@@ -2,34 +2,46 @@
 <section id="venue">
     <div class="content-wrapper">
         <div class="col-lg-10 col-lg-offset-1">
+
             <h3>{{ site.venueBlockTitle }}</h3>
+
+            <div class="col-lg-6 col-lg-offset-3" style="margin-bottom: 60px; padding: 30px; border-style: solid; border-width: 2px; border-color: fff;">
+              <div style="font-size: 24px; line-height: 200%">
+
+                <p style="font-size: 24px; line-height: 200%">We believe that the venue page of this brainhack should reflect the beauty and the diversity of places and people that can be found on our planet.</p>
```

Comment:
- To guide those who might be a bit lost, I would just add at the beginning of the text "All OS-SIG events 2020 will run virtually." so that people undertand why we are doing this page (even if they have not heard about our events becoming virtual).

Reply:
Added a sentence to explain the context.